lib/AST/ASTContext.cpp

```diff
@@ -1683,7 +1683,7 @@ namespace {
       return OrderDeclarations(SrcMgr)(lhs, rhs);
     }
   };
-}
+} // namespace
```

Comment:
Ah, I was matching the rest of your changes, which were dropping the "end". I'm happy to put the "end" in.